examples/tv-casting-app/darwin/MatterTvCastingBridge/MatterTvCastingBridge/templates/MCCommandPayloads.zapt

```diff
@@ -0,0 +1,48 @@
+{{> header excludeZapComment=false}}
+
+#import "MCInteractionModelStructs.h"
+
+NS_ASSUME_NONNULL_BEGIN
+
+@interface MCAbstractPayload : NSObject <NSCopying>
+@end
+
+{{#zcl_clusters}}
+{{#if (isInConfigList name "MatterCastingClusters")}}
+
+// {{asUpperCamelCase name}} cluster:
+{{#zcl_commands}}
+{{#*inline "completeDecl"}}
+{{#if (isSupported cluster command=command isForCommandPayload=true)}}
+
+@interface MC{{cluster}}Cluster{{command}}Params : {{base}}
+{{#zcl_command_arguments}}
+{{#if (isSupported ../cluster command=../command commandField=(asStructPropertyName label))}}
+{{> struct_field_decl cluster=parent.parent.name type=type label=label}};
+{{/if}}
+{{#*inline "oldNameFieldDecl"}}
+{{#if (isSupported ../cluster command=../command commandField=commandField)}}
+{{> struct_field_decl cluster=parent.parent.name type=type label=commandField}};
+{{/if}}
+{{/inline}}
+{{#if (and includeRenamedProperties
+           (hasOldName ../cluster command=../command commandField=(asStructPropertyName label)))}}
```

Comment:
Why do you need the hasOldName bits?  Do you need to promise forever API compatibillity across renames like Matter.framework does?  Or was this just copy/pasted from Matter.framework without really modifying the structure?

Reply:
Yes. It was just copy/pasted from Matter.framework. Removing.